packages/events/src/FederatedMouseEvent.ts

```diff
@@ -1,6 +1,9 @@
 import { Point } from '@pixi/core';
 import { FederatedEvent } from './FederatedEvent';
 
+import type { DisplayObject } from 'pixi.js';
```

Comment:
Maybe `import type { DisplayObject } from '@pixi/display';`? Though circular references in `import type` won't be a big problem.

Reply:
Good catch!